### DISCUSSION TITLE
`Card` or `Tile` concept

### DISCUSSION THREAD

**Discussion by bergser** [SchwarzIT/onyx#2113] [Q&A]
I tried to find something related to the subj. Currently there is only [elevation](https://onyx.schwarz/basics/elevation.html) page which is empty.

In our apps we are using a lot of cards (tiles). Is there any plans to implement this concept or at least any ideas how cards in `onyx` can look like?

**Comment by JoCa96** [comment]
@SergeiZheleznov Thanks for reaching out! A tile is something we want and will have, unfortunately it's not yet planned to be done in the near future. But I created a ticket, that you can track: #2683

If you have specific requirements, you might want to get in contact with @mj-hof or @jannick-ux.

**Comment by JoCa96** [comment]
@bergser Cards and tiles have been implemented 🎉 you can find them here: https://storybook.onyx.schwarz/?path=/docs/basic-card--docs
